bulk/sensor_bulk_load-AssetRecord.csv

```diff
@@ -1608,37 +1608,44 @@ CGINS-METBPR-00263,A01264,Sensor,0,Bulk Meteorology Instrument Package: METBK BP
+CGINS-METCTD-11561,A00715,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-11561,,20140227,5816,,Woods Hole Oceanographic Institution
+CGINS-METCTD-12754,A01266,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-12754,,20141214,5816,,Woods Hole Oceanographic Institution
+CGINS-METCTD-12755,A01267,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-12755,,20141214,5816,,Woods Hole Oceanographic Institution
+CGINS-METCTD-12756,A01268,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-12756,,20141214,5816,,Woods Hole Oceanographic Institution
+CGINS-METCTD-12757,A01269,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-12757,,20141214,5816,,Woods Hole Oceanographic Institution
+CGINS-METCTD-12758,A01270,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-12758,,20141214,5816,,Woods Hole Oceanographic Institution
+CGINS-METCTD-13468,A01608,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-13468,,,,,Oregon State University
+CGINS-METCTD-13469,A01609,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-13469,,,,,Oregon State University
+CGINS-METCTD-13470,A01858,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-13470,,,,,Oregon State University
+CGINS-METCTD-13471,A01859,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-13471,,,,,Oregon State University
+CGINS-METCTD-13472,A01860,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-13472,,,,,Woods Hole Oceanographic Institution
 CGINS-METCTD-13542,A01861,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-13542,,,,,Woods Hole Oceanographic Institution
 CGINS-METCTD-13543,A01862,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-13543,,,,,Woods Hole Oceanographic Institution
+CGINS-METCTD-13544,A01863,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-13544,,,,,Woods Hole Oceanographic Institution
 CGINS-METCTD-13545,A01864,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-13545,,,,,Woods Hole Oceanographic Institution
```

Comment:
For 13545, add Comment "Lost at sea on GI01SUMO-00004".

Reply:
done